utils/publish.sh

```diff
@@ -5,24 +5,16 @@ set -euo pipefail
 # can't publish to. Unset so that the default npm registry is used.
 unset npm_config_registry
 
-__dirname="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-echo "__dirname: ${__dirname}"
-
 if [ -z "$NPM_TOKEN" ]; then
   echo "NPM_TOKEN not found. Did you forget to assign the GitHub Action secret?"
   exit 1
 fi
 
 echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
 
-if [ ! -e ~/.npmrc ]; then
```

Comment:
Removed this block because there's no possibility that `~/.npmrc` does not exist, since it's being written to in the line above.